app/views/navbar.scala.html

```diff
@@ -245,19 +231,44 @@ <h4 class="modal-title" id="sign-up-label">Sign up</h4>
                     }
                 });
 
-                $("[data-hide]").on("click", function(){
-                    $("." + $(this).attr("data-hide")).hide();
-                    // -or-, see below
-                    // $(this).closest("." + $(this).attr("data-hide")).hide();
+                //set tooltip alerts for when users do not enter all sign in / sign up fields
```

Comment:
This is old code. @sbower213 can you make sure this file has the proper email validation that was updated in https://github.com/ProjectSidewalk/SidewalkWebpage/pull/883/files